lib/kubernetes-deploy/kubernetes_resource.rb

```diff
@@ -192,5 +193,10 @@ def pretty_status
     def kubectl
       @kubectl ||= Kubectl.new(namespace: @namespace, context: @context, logger: @logger, log_failure_by_default: false)
     end
+
+    def template
```

Comment:
Why is this needed, and why is it okay to return `nil` if the file is not present?